services/motion/builtin/builtin_test.go

```diff
@@ -234,8 +234,6 @@ func TestMoveWithObstacles(t *testing.T) {
 }
 
 func TestMoveOnMapLongDistance(t *testing.T) {
-	// TODO(RSDK-6326) - fix test failure and unskip
```

Comment:
Recent updates have this test running extremely fast and it should not time out

Reply:
Awesome!